README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world-python
 Learning to Python
+
+Hello Barney. I'm all up in your repo
```

Comment:
You can also leave comments on specific lines of code. Very handy for making clear exactly what you're talking about